Improve UI of About Us page section

Anubhav-Pal opened the pull request:
## Issue Addressed
Improved the UI design of About us section that before was unappealing.


![FARMHUB](https://github.com/marshadkhn/Farmhub/assets/95634562/6a49f30c-7b6e-4ad8-8f4c-1d76fe40a85f)


## Screenshots

<!-- Include any relevant screenshots or images that illustrate the changes you've made. Screenshots can help reviewers and users understand the visual impact of your changes. -->

Abdul-365 reviewed (approved):
Much better!

Abdul-365 commented:
I think the cards in services should be a little bigger. @marshadkhn What do you think?

Shriansh2002 reviewed (approved):
It looks nice! 👍

Shriansh2002 commented:
> I think the cards in services should be a little bigger. @marshadkhn What do you think?

Yes, @Abdul-365 You are right, it can be a bit bigger

marshadkhn commented:
@Anubhav-Pal Just make the cards a bit bigger and clean 
Did a great job

Anubhav-Pal commented:
Hey, I've made the requested changes in this PR. The cards are now aligned with the maintainers cards section. The code addresses the issues raised in #91. Please review the updates when you have a chance. Thank you!. 

Here's the ss of updated UI: 

![FARMHUB](https://github.com/marshadkhn/Farmhub/assets/95634562/4fc5b6e8-03aa-4aa0-8390-36b2ce726339)

marshadkhn commented:
@Abdul-365 @Shriansh2002 
review this PR
